neo/public/javascripts/timeseries/controllers.js

```diff
@@ -102,9 +143,9 @@ angular.module('cloudberry.timeseries', ['cloudberry.common'])
                 .text('Reset')
                 .attr('href',"#")
                 .on("click", function() { timeSeries.filterAll(); dc.redrawAll(); requestFunc(minDate, maxDate);})
-                .style("position", "inherit")
+                .style("position", "absolute")

```

Comment:
Absolute: The element is positioned relative to its first positioned (not static) ancestor element. 
Inherit: Inherits the position property from its parent element.
So in this case, make the "a" tag's position absolute will make its positioning properties like "left", "bottom" be relative to the time-series tag whose position is set to relative. If I don't do this, the reset button on the time series will be in some strange place.